documentation/course/E1.md

```diff
@@ -113,7 +117,27 @@ W zakres projektu wchodzi:
 
 ### Płatność (Piotr Kot)
 
-- `PAY/01` - TODO@piterek130 - historyjki użytkownika
+- `PAY/01` - Jako _pasażer_ chcę mieć możliwość wyświetlenia listy dostępnych metod płatności.
+- `PAY/02` - Jako _pasażer_ chcę mieć możliwość dodania metody płatności BLIK do swojego _konta_.
+- `PAY/03` - Jako _pasażer_ chcę mieć możliwość dodania nowej karty płatniczej.
+- `PAY/04` - Jako _pasażer_ chcę mieć możliwość usunięcia karty płatniczej ze swojego _konta_.
+- `PAY/05` - Jako _pasażer_ chcę mieć możliwość usunięcia metody płatności ze swojego _konta_.
+- `PAY/06` - Jako _pasażer_ chcę mieć możliwość wyboru BLIK jako metodę płatności.
+- `PAY/07` - Jako _pasażer_ chcę mieć możliwość wyboru karty płatniczej jako metodę płatności. 
```

Comment:
I'd add the following terms to the dictionary and emphasise them here:
- `BLIK`
- `Dane karty kredytowej` (no idea why it wasn't in the dictionary, since it's in the class diagram)